README.md

```diff
@@ -801,15 +801,15 @@ See the [AUTHORS][] file for author contact information.
 License
 -------
 
-Bedrock and all Bedrock modules are:
+Bedrock core libraries are licensed under the [Apache-2.0](LICENSE).
```

Comment:
This updated text also is not explicit about the license of *this* project.  It can be inferred that this is a "Bedrock core library" but could be direct and say "This library and some other Bedrock core libraries are ..." to make it clear.  (I'm not sure we have defined "core" yet so maybe phrase with "some" such that people will check.)

Reply:
Would prefixing that section with the following do the trick?
```suggestion
[Apache-2.0](LICENSE) Copyright YYYY-YYYY Digital Bazaar, Inc.

Bedrock core libraries are licensed under the [Apache-2.0](LICENSE).
```

"Project licensing" is basically an assumption about per-file licensing. Moving to implementing a more verbose description format (in this case REUSE v3.0), we avoid that assumption (with its common misapplications to other people's software) and move to file-by-file copyright and license claims--to avoid confusion.

That said, I agree it's a common signal in the README, so adding that seems reasonable.